### PR TITLE
Fix wiki challenge solution template

### PR DESCRIPTION
**Pull request by abhisekp**
Also fixed related files which uses the template

Rename file Challenge Solution Template to add Wiki namespace
- Wiki specific/related files must start with "Wiki-" file name.

Fix Challenge Solution Template markdown
- Hints and other explanations should be properly legible and written clearly.
- Fix the blockquoted hints and explanations.

----
#### Related Files
Fix Algo Caesar Cipher markdown
- Reformat in accordance with the Wiki Challenge Solution Template

Fix Algo Largest of Number markdown format
- In accordance with Wiki Challenge Solution Template

Fix Algo Reverse String Markdown
- In accordance with Wiki Challenge Solution Template

----
@FreeCodeCamp/wiki-mods Use normal merge. **DO NOT SQUASH**

### CODE REVIEW

**Comment by alayek** [comment]
# LGTM :+1: 

Let @Rafase282 review this as well.

File: Algorithm-Caesars-Cipher.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
-:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program. :busts_in_silhouette: Write your own code. :memo:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: . Write your own code :memo: .
```
Comment (Rafase282) [inline Algorithm-Caesars-Cipher.md]: I would put the icon at the beginning and the dot at the end. It doesn't looks good the way it is with the dot after the icon. Too much space.

File: Algorithm-Caesars-Cipher.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
 
 # :checkered_flag: Problem Explanation:
-> - You need to write a function, which will take string encoded with
+- You need to write a function, which will take string encoded with
```
Comment (Rafase282) [inline Algorithm-Caesars-Cipher.md]: ... which will take **a** string encoded ...

**Comment by Rafase282** [comment]
I pointed out some changes.

File: Algorithm-Caesars-Cipher.md (hunk 2 above)
Comment (abhisekp) [inline Algorithm-Caesars-Cipher.md]: This change can be done in another PR. This PR is dedicated to markdown formatting changes in the template file and it's related files only.

Comment (Rafase282) [inline Algorithm-Caesars-Cipher.md]: Alright, let me know when you update the markdown so I can make that PR after this one is merged.

**Comment by abhisekp** [comment]
Everything else intact, only the following header part of every file is updated.
```md
:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :memo:
```
Hence, no need for checking other parts of the modified files.

**Pull request merged**